.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,75 @@
+name: Release Artifacts
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+**'
+    branches:
+      - 'release/**'
+      - 'develop'
+      - 'plugin_ci'
+
+jobs:
+  kubectl-plugin:
+    if: startsWith(github.ref, 'refs/tags/v4.3')
```

Comment:
This cannot be like this, otherwise push to branches won't build

Reply:
We will have to remove this eventually..

Reply:
added the check to publish-release-artifacts.yaml